wpilibc/athena/src/Threads.cpp

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "HAL/Threads.h"
```

Comment:
It is. Adding NOLINT won't fix it either because they are both part of the same header group, `related`, and NOLINT makes includes marked with it part of `related`. Perhaps NOLINT should maintain an include's order with respect to those around it, but that wouldn't work if they're all in the wrong order to start.